.github/pull_request_template.md

```diff
@@ -1,58 +1,19 @@
-Please use the following template to submit a pull request, filling
-in at least the "Text for release notes" and/or "Further details".
-Thank you!
+# Summary
```

Comment:
I am not sure this headline is a good idea, we could also just drop it...

Reply:
Happy to get rid of it and include 'here' in the first sentence.